gulpfile.js

```diff
@@ -199,7 +199,15 @@ gulp.task(
   gulp.series('sprites-clean', () =>
     gulp
       .src(`${paths.ICONS_DIR}**/*.svg`)
-      .pipe(svgmin())
+      .pipe(
+        svgmin({
+          plugins: [
+            {
+              removeDimensions: false,
+            },
+          ],
+        }),
+      )
```

Comment:
@mirisuzanne Ok, so the advice I gave you was incorrect, and this isn't doing anything. I'm not sure what's the practical impact of having height/width explicitly defined, but I'll push it in a minute.

Reply:
Hm, I'm not actually sure what the goal is here. The SVGs are combined into a spritesheet with one `svg` containing multiple `symbol`s... but I don't think `height` and `width` are even valid attributes for a `symbol` element. Maybe we want `height`/`width` on the `use` element instead?